js/scorm/wrapper.js

```diff
@@ -364,28 +368,19 @@ class ScormWrapper {
   }
 
   recordInteraction(id, response, correct, latency, type) {
-    if (!this.isSupported('cmi.interactions._count')) {
-      this.logger.info('ScormWrapper::recordInteraction: cmi.interactions are not supported by this LMS...');
-      return;
-    }
-
     switch (type) {
       case 'choice':
-        this.recordInteractionMultipleChoice.apply(this, arguments);
+        this.recordInteractionMultipleChoice(...arguments);
```

Comment:
I don't think we could spread the arguments object because it isn't iterable.

https://caniuse.com/mdn-javascript_functions_arguments_--iterator
https://stackoverflow.com/questions/34614653/is-the-arguments-object-supposed-to-be-an-iterable-in-es6

Apparently one can now. Cools.